model/accounts_mgmt/v1/roles_resource.model

```diff
@@ -32,6 +32,23 @@ resource Roles {
 		// regardless of the size of the page.
 		in out Total Integer
 
+		// Search criteria.
+		//
+		// The syntax of this parameter is similar to the syntax of the _where_ clause
+		// of an SQL statement, but using the names of the attributes of the role
+		// instead of the names of the columns of a table. For example, in order to
+	        // retrieve roles named starting with Organization:
```

Comment:
Put back quotes around `Organization`, that way it will be rendered (in the future) using fixed a fixed with font.